tests/test_libraries.py

```diff
@@ -131,6 +131,32 @@ def test_library_add(caplog):
         in caplog.text
     )
 
+    tcf = tempfile.NamedTemporaryFile(mode="w+")
+    args.config = tcf
+
+    vars(args)["sync-type"] = "git"
+    vars(args)["sync-uri"] = sync_uri
+    vars(args)["sync-version"] = "capi2"
+    args.location = None
+
+    expected = """[library.fusesoc-cores]
+location = fusesoc_libraries/fusesoc-cores
+sync-uri = https://github.com/fusesoc/fusesoc-cores
+sync-version = capi2
```

Comment:
This isn't super nice as the test will fail if this branch doesn't exist, wasn't sure of the best approach as the test clones the repository